seed/schema_data.sql

```diff
@@ -1,188 +1,25 @@
--- MySQL dump 10.13  Distrib 5.7.29, for osx10.14 (x86_64)
---
--- Host: localhost    Database: adidas
--- ------------------------------------------------------
--- Server version	5.7.29
+DROP DATABASE IF EXISTS adidas;
+
+CREATE DATABASE adidas;
+
+USE adidas;
+
+DROP TABLE IF EXISTS categoryInfo;
```

Comment:
don't need to drop table when you're already dropping the database

Reply:
done